ext/standard/basic_functions.c

```diff
@@ -2624,3 +2624,23 @@ PHP_FUNCTION(sys_getloadavg)
 }
 /* }}} */
 #endif
+
+PHP_FUNCTION(num_cpus)
+{
+	ZEND_PARSE_PARAMETERS_NONE();
+
+	#if defined(_SC_NPROCESSORS_ONLN)
```

Comment:
Just NIT: the preprocesor macros are not usually indented in our code so if you could remove indent for them, it would be great.

Reply:
Sure, they are much easier to spot when not indented.